Makefile

```diff
@@ -1,3 +1,21 @@
+build:
```

Comment:
I see a number of things like this from other already-merged PRs, so I wonder if this could use a thorough rebase over the latest master, but it probably won't matter. It does make it a bit harder to read through, however. What do you think?

Reply:
yeah this happened because I merged my PR with the master. Should I try a thorough rebase?

Reply:
Yes perhaps! However try to address the other issues first since we'll lose our comments and discussion I believe.

Reply:
Yes perhaps! However try to address the other issues first since we'll lose our comments and discussion I believe.